src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -605,9 +486,9 @@ static void Main()
         (int, int, int, int, int, int, int, int) y = (1, 2, 3, 4, 5, 6, 7, 8, 9);
     }
 }
-" + trivial2uple + trivial3uple + trivalRemainingTuples;
+";
 
-            CreateCompilationWithMscorlib(source, parseOptions: TestOptions.Regular.WithTuplesFeature()).VerifyDiagnostics(
+            CreateCompilationWithMscorlib(source, references: new[] { ValueTupleRef, SystemRuntimeFacadeRef }, parseOptions: TestOptions.Regular.WithTuplesFeature()).VerifyDiagnostics(
```

Comment:
Good question. `System.ValueTuple` has a dependency on `System.Runtime` because the corefx resource manager depends on it.

Here are the current references in the assembly:
```IL
.assembly extern System.Runtime
.assembly extern System.Resources.ResourceManager
.assembly extern System.Collections
.assembly extern System.Diagnostics.Debug
```